RadarSDK/RadarLocationManager.m

```diff
@@ -411,12 +451,82 @@ - (void)updateTrackingFromMeta:(RadarMeta *_Nullable)meta {
 
 }
 
+
+// Call this method when the tracking state changes
+- (void)changeRampedState:(BOOL)rampedUp {
```

Comment:
Things I'm thinking about:
1) Should we just not support a timeout in v1 to avoid the complexity of `StateChange`?
2) Can we get away with a single variable to track the running total of time spent ramped up instead of a list of `StateChange`s?
3) Should we make the timeout configurable on tracking options, something like 'nearbyTrackingTimeout`?